covasim/people.py

```diff
@@ -312,32 +313,11 @@ def make_people(sim, verbose=None, id_len=None, die=True, reset=False):
 
     return
 
-def make_randpop(sim, id_len=6):
+def make_randpop(sim, id_len=6, country="United States of America"):
     ''' Make a random population, without contacts '''
 
     # Load age data based on 2018 Seattle demographics
-    age_data = np.array([
-        [ 0,  4, 0.0605],
-        [ 5,  9, 0.0607],
-        [10, 14, 0.0566],
-        [15, 19, 0.0557],
-        [20, 24, 0.0612],
-        [25, 29, 0.0843],
-        [30, 34, 0.0848],
-        [35, 39, 0.0764],
-        [40, 44, 0.0697],
-        [45, 49, 0.0701],
-        [50, 54, 0.0681],
-        [55, 59, 0.0653],
-        [60, 64, 0.0591],
-        [65, 69, 0.0453],
-        [70, 74, 0.0312],
-        [75, 79, 0.02016], # Calculated based on 0.0504 total for >=75
-        [80, 84, 0.01344],
-        [85, 89, 0.01008],
-        [90, 99, 0.00672],
-        ])
-
+    age_data = load_country_pop(country)
```

Comment:
This is cool, I like how simple this is!

Reply:
There is another call for the ui's which will return all the available countries as well.